src/jobs/models/jobManager.js

```diff
@@ -298,11 +299,21 @@ function addCron(jobId, job, cronExpression, configData) {
 async function globalWebhookAssignmentGuard(webhookIds) {
```

Comment:
rename function to `validateWebhookAssignment` since it's not only related to global webhooks anymore